js/console.ts

```diff
@@ -272,20 +282,26 @@ function createRawObjectString(
   }
   ctx.add(value);
 
-  const entries: string[] = [];
   let baseString = "";
 
   const className = getClassInstanceName(value);
   let shouldShowClassName = false;
   if (className && className !== "Object" && className !== "anonymous") {
     shouldShowClassName = true;
   }
-
-  for (const key of Object.keys(value)) {
-    entries.push(
-      `${key}: ${stringifyWithQuotes(value[key], ctx, level + 1, maxLevel)}`
-    );
-  }
+  const keys = Object.keys(value);
+  const entries: string[] = keys.map(key => {
+    if (keys.length > OBJ_ABBREVIATE_SIZE) {
```

Comment:
it works, it reads kinda inside out to me.
Maybe you can just do this:
```
let keys = Object.keys(value);
if (keys.length < OBJ_ABBREVIATE_SIZE) {
  keys = keys.map(k => ........);
}
return keys;

Reply:
Noted - but I think I like my version better.